pandas/tests/groupby/test_any_all.py

```diff
@@ -83,6 +83,25 @@ def test_any_non_keyword_deprecation():
     tm.assert_equal(result, expected)
 
 
+def test_any_apply_keyword_non_zero_axis_regression():
+    # https://github.com/pandas-dev/pandas/issues/48656
+    df = DataFrame({"A": [1, 2, 0], "B": [0, 2, 0], "C": [0, 0, 0]})
+    expected = Series([True, True, False])
+    tm.assert_series_equal(df.any(axis=1), expected)
+
+    result = df.apply("any", axis=1)
+    tm.assert_series_equal(result, expected)
+
+    msg = (
+        "In a future version of pandas all arguments of "
+        "DataFrame.any and Series.any will be keyword-only."
+    )
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        expected = df.any(1)
```

Comment:
Sorry that I wasn't in clear, in 

https://github.com/pandas-dev/pandas/pull/48797#discussion_r982130587

I meant to construct `expected` explicitly here as well. In fact, it's the same `expected` as above, right? Can we just drop all these highlighted lines of code?

`DataFrame.any` is already tested (with or without keyword arguments), it's `DataFrame.apply("any", 1)` and `DataFrame.apply("any", axis=1)` that need testing